maggy/core/experiment_driver/optimization_driver.py

```diff
@@ -89,13 +92,57 @@ def __init__(self, config, num_executors, log_dir):
         self.controller.direction = self.direction
         self.controller._initialize(exp_dir=self.log_dir)
 
-    def _register_callbacks(self):
+    def _exp_startup_callback(self):
+        tensorboard._write_hparams_config(
+            EnvSing.get_instance().get_logdir(self.APP_ID, self.RUN_ID),
+            self.config.searchspace,
+        )
```

Comment:
@RiccardoGrigoletto is this something that should be in the environment?

Reply:
@moritzmeister I think it's fine to have it here.